orocos_kdl/src/chainjnttojacdotsolver.cpp

```diff
@@ -0,0 +1,262 @@
+/*
+    Computes the Jacobian time derivative
+    Copyright (C) 2015  Antoine Hoarau <hoarau [at] isir.upmc.fr>
+
+    This library is free software; you can redistribute it and/or
+    modify it under the terms of the GNU Lesser General Public
+    License as published by the Free Software Foundation; either
+    version 2.1 of the License, or (at your option) any later version.
+
+    This library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public
+    License along with this library; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+
+#include "chainjnttojacdotsolver.hpp"
+#include "chainfksolverpos_recursive.hpp"
+#include "chainjnttojacsolver.hpp"
+//#include "kinfam_io.hpp"
+//#include "frames_io.hpp"
+//#include "framevel_io.hpp"
+
+namespace KDL
+{
+
+inline void Skew(Rotation& R,const Vector& sk)
+{
+    R(0,0) = 0.0;
+    R(0,1) = - sk.z();
+    R(0,2) =   sk.y();
+    R(1,0) =   sk.z();
+    R(1,1) = 0.0;
+    R(1,2) = - sk.x();
+    R(2,0) = - sk.y();
+    R(2,1) =   sk.x();
+    R(2,2) = 0.0;    
+}
```

Comment:
This only seems to be used for taking a cross product of two `Vector`s `a` and `b`. In KDL we can simply use `a * b` for this.